repose-aggregator/artifacts/src/config/scripts/repose-valve-systemd

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+command -v "$JAVA_CMD" >/dev/null 2>&1
+if [ $? -gt 0 ]; then
+    echo 'Repose requires the "JAVA_CMD" environment variable be set to the absolute path of a Java executable!'
+    echo " - The current value ($JAVA_CMD) is invalid."
+    echo ' - This can be updated in the configuration file: /etc/systemd/system/repose-valve.service.d/local.conf'
+    exit 1
+fi
+
+VERSION=$("$JAVA_CMD" -version 2>&1 | grep ' version' | awk '{ print substr($3, 2, length($3)-2); }')
+JAVA_MINOR=$(echo $VERSION | tr "." " " | cut -d " " -f2)
+if [ "$JAVA_MINOR" -lt "8" ]; then
+    echo 'Repose requires a Java version of at least 8 to function.'
+    echo 'Please install a JRE 1.8 or greater.'
+    exit 2
+else
+    echo 'Java is at JRE 1.8 or greater.'
```

Comment:
I'm curious why you print something out for the success case here.  It doesn't hurt anything, just curious.